frontend/src/js/tooltip/reducer.ts

```diff
@@ -22,6 +22,7 @@ export type AdditionalInfosType = {
   matchingEntities: number | null;
   dateRange?: DateRangeT;
   infos?: InfoType[];
+  parent?: string | null;
```

Comment:
any reason to allow null **and** undefined here? How about just
```suggestion
  parent?: string
```